### PR TITLE
Compute expansion options for additional created groups

### PR DESCRIPTION
**Pull request by aleksandra-malinowska**
Compute expansion options for additional created groups.

(I'm working on a test that would have caught the bug, TBA in another commit to this PR.)

### CODE REVIEW

File: cluster-autoscaler/core/scale_up.go [hunk 1]
```diff
@@ -467,6 +475,12 @@ func ScaleUp(context *context.AutoscalingContext, processors *ca_processors.Auto
 					continue
 				}
 				nodeInfos[nodeGroup.Id()] = nodeInfo
+
+				option := computeExpansionOption(context, podEquivalenceGroups, nodeGroup, nodeInfo, upcomingNodes)
+
+				if len(option.Pods) > 0 && option.NodeCount > 0 {
```
Comment (losipiuk) [inline cluster-autoscaler/core/scale_up.go]: Do we need this if? Why not put it always in the map. Even if pods are empty. No harm, right?

Comment (losipiuk) [reply cluster-autoscaler/core/scale_up.go]: Edit - maybe this is safer. We are checking for size of expansion options later. Keep it.

**Comment by aleksandra-malinowska** [comment]
Added commit with test - fails before the fix, passes with the fix.

**Pull request merged**